Apps/W1/HybridGP/app/src/Migration/History/GPPopulateHistTables.Codeunit.al

```diff
@@ -947,12 +949,13 @@ codeunit 40900 "GP Populate Hist. Tables"
         HistMigrationStatusMgmt.UpdateStepStatus("Hist. Migration Step Type"::"GP Purchase Receivables Trx.", true);
     end;
 
-    local procedure PopulatePurchaseRecvLines(GPPOPReceiptHist: Record GPPOPReceiptHist; ReceiptNo: Code[35])
+    local procedure PopulatePurchaseRecvLines(var GPPOPReceiptHist: Record GPPOPReceiptHist; ReceiptNo: Code[35])
```

Comment:
Do we want the procedure to be able to modify the record? I thought 'var' was to be used in only those cases where it is ok, or expected, that the record gets updated.

Reply:
The use of var here is part of the performance improvements. It passes the record by reference, while without it, it has to create another copy of the record in memory before passing it as a parameter.

Reply:
Correct, passing by reference is faster but also allows for the procedure to modify the object whereas creating another copy leaves the original unchanged. If a possible modification is not an issue 'var' is fine.